.travis.yml

```diff
@@ -27,11 +27,11 @@ install:
   - pip install pytest pytest-cov coveralls
   - pip install flake8 black
   - pip install mypy
-  - pip install .$DEP
+  - pip install -e .$DEP
 script:
   - black --check .
   - flake8 --config=.flake8 .
   - mypy cgp
-  - cd test && pytest --cov=cgp
+  - pytest test --cov=cgp
```

Comment:
do you need the `test` argument here? shouldn't pytest automatically discover the tests there?

Reply:
no, not necessary, removed, thanks.